### DISCUSSION TITLE
如何修改元数据nfo文件的命名方式

### DISCUSSION THREAD

**Discussion by KuanKan410** [metatube-community/jellyfin-plugin-metatube#387] [General]
如题，目前元数据的命名方式似乎是"movie.nfo"，如何修改配置使其成为对应的文件名"XXXX-1234.nfo"的样式呢？

**Comment by xjasonlyu** [comment]
我不记得有什么元数据命名成.nfo的，至少不是插件的行为。

**Reply by KuanKan410** [reply]
不好意思，没描述清楚，在媒体库中勾选了储存为nfo格式的选项

**Comment by KuanKan410** [comment]
疑似是jellyfin的问题，正常的电影使用刮削也有时会出现被命名为"movie.nfo"的情况

**Comment by nj106** [comment]
> 如题，目前元数据的命名方式似乎是"movie.nfo"，如何修改配置使其成为对应的文件名"XXXX-1234.nfo"的样式呢？

有个讨巧方法：把所有小姐姐视频，全部放在同一个目录里。

**Comment by kyleflower** [comment]
有没有解决方法，极空间识别需要和影片文件名一致
